### ISSUE TITLE
[request] vtu11/1.2.0

### ISSUE THREAD

**Issue by jellespijker** [conan-io/conan-center-index#14802]
### Package Name/Version

vtu11/1.2.0

### Webpage

https://github.com/phmkopp/vtu11

### Source code

https://github.com/phmkopp/vtu11

### Description of the library/tool

Vtu11 is a small C++ header-only library to write unstructured grids using the vtu file format. It keeps the mess of dealing with file writing in different formats away from you. Currently it does not add any features for setting up the required data structure because this vastly differs based on the context in which vtu11 is used.

**Comment by jellespijker** [comment]
PR to follow

**Comment by jellespijker** [comment]
The PR passes all tests, now waiting on 3 reviewers.

**Comment by jellespijker** [comment]
It looks like the PR was merged, @SpaceIm made an astute observation however about the patch being dependent on the C++ std version, while the package_id is cleared. I will open up a new PR with a proposed fix for this somewhere next week.
